test/e2e/tests/test_transformjob.py

```diff
@@ -23,142 +23,165 @@
 from e2e import (
     service_marker,
     create_sagemaker_resource,
+    wait_for_status,
 )
 from e2e.replacement_values import REPLACEMENT_VALUES
 from e2e.bootstrap_resources import get_bootstrap_resources
+from e2e.common.config import *
+from time import sleep
 
-RESOURCE_PLURAL = "transformjobs"
+RESOURCE_PLURAL = 'transformjobs'
 
+def _make_model():
```

Comment:
- make this a fixture so you add delete in case the test fails
- if possible use the model name like `transform_resource_name+"-model"` so its easy to identify while debugging

Reply:
I did the second for which I had to pass the model name as a parameter. 
Now when I make the make_model a fixture, I cannot use it as one since it needs a parameter. So I am using a fixture as a regular method, unless I want to make the transformjob name global.

Reply:
Dont forget you clean up the model in transform fixture